python/pyspark/context.py

```diff
@@ -71,6 +71,7 @@ class SparkContext(object):
     _active_spark_context = None
     _lock = RLock()
     _python_includes = None  # zip and egg files that need to be added to PYTHONPATH
+    _thread_pool_executor = None
```

Comment:
Can we give this a more specific name to avoid confusing with things that are on the workers?